### ISSUE TITLE
Amazon Pay for Alexa + ブロックチェーンを使った寄付アプリを実装！

### ISSUE THREAD

**Issue by xiaojcai** [aws-events/aws-dev-day-online-japan-2021-cfp#4]
### イベント登録規約、AWS 行動規範 Code of Conduct への同意 (必須)

- [X] 私は、[イベント登録規約](https://aws.amazon.com/jp/events/terms/)、および [AWS 行動規範 Code of Conduct](https://aws.amazon.com/jp/codeofconduct/) を確認し、同意しました

### Twitter ID (必須)

@johna1203

### セッションタイトル (必須)

- [X] セッションのタイトルをイシュー件名に最大40文字程度で入力しました

### セッションのアブストラクト (最大250文字) (必須)

驚くほどシンプルな1-Yes音声寄付のAlexaスキルをご紹介します。このAlexaスキルでは、Amazon Payを利用することで、 個人情報、クレジットカード情報を一切入力せずに音声だけで寄付できるフローを実現します。さらに、Amazon Managed Blockchain上でEthereum ネットワークのノードを作り、ライブプログラムでSolidityでスマートコントラクト部分を作成します。Amazon Payで寄付された額をブロックチェーンで管理するようなシステムをLiveでお見せします！

### セッションについての補足情報 (最大800文字) (任意)

参加者：複数登壇予定です。

### 想定受講者の知識レベル（必須）

初級 - セッションの中心となるトピックについての具体的な知識がない方、これから勉強しようと考えている方向け

### 想定受講者の開発対象やロール・役割 (複数選択可) (必須)

Web バックエンド / サーバーサイド開発

### 想定受講者の業種・業界・業態 (複数選択可) (任意)

エンタープライズ, 金融

### セッションのトピック (複数選択可) (必須)

プログラミング

### セッションの技術カテゴリー (複数選択可) (必須)

サーバーレス, ブロックチェーン

### セッション内で登場する主な AWS サービス (任意)

Amazon Managed Blockchain, Lambda, API Gateway

**Comment by johna1203** [comment]
> AWS Dev Day Online Japan 2021 へのセッション応募ありがとうございます！
> 
> セッション審査は本イシューへの GitHub リアクション、Twitter での反響も加味して実施します。[運営によるツイート](https://twitter.com/toricls/status/1423433504977653765)のリツイートや、あるいはご自身での宣伝など、ぜひぜひソーシャルでのシェアもお願いいたします。
> 
> 採択の場合はご記入いただいた Twitter ID へのダイレクトメッセージにて運営よりご連絡しますので、[Twitter DM の受け取り設定](https://help.twitter.com/ja/using-twitter/direct-messages#receive)の確認もあわせてお願いいたします。

はい、TwitterのDM受け取り設定済みです。SNSに投稿してできるだけ拡散していきたいと思いますので、よろしくお願いします。
